satpy/etc/readers/modis_l2.yaml

```diff
@@ -86,3 +93,103 @@ datasets:
     file_type: mod35_hdf
     coordinates: [longitude, latitude]
 
+##########################
+#Datasets in file mod06_l2
+##########################
+  cloud_fraction:
+    name: cloud_fraction
+    resolution: 5000
+    file_key: Cloud_Fraction
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  brightness_temperature:
+    name: brightness_temperature
+    resolution: 5000
+    file_key: Brightness_Temperature
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  surface_temperature:
+    name: surface_temperature
+    resolution: 5000
+    file_key: Surface_Temperature
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  surface_temperature_1km:
+    name: surface_temperature_1km
+    resolution: 1000
+    file_key: surface_temperature_1km
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  surface_pressure:
+    name: surface_pressure
+    resolution: 5000
+    file_key: Surface_Pressure
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  cloud_top_height:
+    name: cloud_top_height
+    resolution: 5000
+    file_key: Cloud_Top_Height
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  cloud_top_height_1km:
+    name: cloud_top_height_1km
+    resolution: 1000
+    file_key: cloud_top_height_1km
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  cloud_top_temperature:
+    name: cloud_top_temperature
+    resolution: 5000
+    file_key: Cloud_Top_Temperature
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  cloud_top_temperature_1km:
+    name: cloud_top_temperature_1km
+    resolution: 1000
+    file_key: cloud_top_temperature_1km
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  cloud_effective_emissivity:
+    name: cloud_effective_emissivity
+    resolution: 5000
+    file_key: Cloud_Effective_Emissivity
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  cloud_effective_emissivity_1km:
+    name: cloud_effective_emissivity_1km
+    resolution: 5000
+    file_key: cloud_emissivity_1km
+    file_type: mod06_hdf
+    coordinates: [longitude, latitude]
+
+  cloud_effective_radius_1km:
+    name: cloud_effective_radius_1km
```

Comment:
Why does this name have the resolution in it when the others don't?

Reply:
I am not sure right now if this is true for that specific dataset but some of them have 5km and 1km datasets in the file and are named that way then too. I will check up on that. Eventually if only 1km is present the 1km can be removed from the name.

Reply:
I am not sure right now if this is true for that specific dataset but some of them have 5km and 1km datasets in the file and are named that way then too. I will check up on that. Eventually if only 1km is present the 1km can be removed from the name.